Show localized news date in title for all news items

Nachtalb opened the pull request:
Issue for this PR: https://github.com/4teamwork/bern.web/issues/1325

The date is shown for all Content Types which have set the start index. This includes eg. `Event` from plone or `ftw.news.News`

Eg. this would look like: "Some News Title (Oct 04, 2017)" or when you search for an item "Some News Title (Oct 04, 2017) (/path/to/item)"

maethu commented on ftw/referencewidget/browser/search.py:
```diff
@@ -45,15 +46,20 @@ def __call__(self):
 
-            label = '{0} ({1})'.format(item.Title, item.getPath())
+            date = ''
+            if item.portal_type == 'ftw.news.News':
```
@Nachtalb 
Please remove the dependency to ftw.news and check the brain for the `start` attribute.
Show content of the start attr if there is any value

Nachtalb replied:
Do you mean like this? 
```python
date = ' (%s)' %  item.start if item.start else ''
```

maethu replied:
Or something similar, you may use  `getattr`

Nachtalb replied:
> you may use `getattr`

Doesn't every brain have a index if you add a new one. The only difference would be it it has a value or not, or am I wrong? That would mean that the getattr doesn't give any advantage. And I can't just use a default value like '' for the start attribute, because I still have to add the `' (%s)'` __only__ if it has a value. 

Also I just realized I would still have to Localize it.
```Python
date = ' (%s)' % plone.toLocalizedTime(item.start) if item.start else '' 
```

maethu replied:
yep go ahead 😄

Nachtalb commented:
@maethu 
- added requested date changes
- changed commit message from `of all \`ftw.new.News\` items` to `for all news items`
- added changelog entry as 1.5.0

maethu commented on setup.py:
```diff
@@ -10,6 +10,8 @@
     'unittest2',
     'ftw.testbrowser >= 1.26.1',
+    'ftw.testing',
+    'ftw.news'
```
@Nachtalb Can we get rid of the ftw.news test dep.?
We could use the plone default news type or our own DX type?

Nachtalb commented:
@maethu removed ftw.news.News and test now is made with Plone Events

maethu commented on ftw/referencewidget/tests/builders.py:
```diff
@@ -7,3 +8,9 @@ class SampleContentBuilder(DexterityBuilder):
     'refwidget sample content', SampleContentBuilder)
+
+
+class NewsItemBuilder(ArchetypesBuilder):
```
@Nachtalb Well... I now this is petty... can you fix the naming. Everything else 👍

Nachtalb replied:
Sry for that, done 👍
